server/server_signals_windows.go

```diff
@@ -0,0 +1,26 @@
+// +build windows
+
+package server
+
+import (
+	"os/signal"
+	"syscall"
+
+	"github.com/containous/traefik/log"
+)
+
+func (server *Server) configureSignals() {
+	signal.Notify(server.signals, syscall.SIGINT, syscall.SIGTERM)
+}
+
+func (server *Server) listenSignals() {
+	for {
+		sig := <-server.signals
+		switch sig {
+		default:
+			log.Infof("I have to go... %+v", sig)
```

Comment:
Can we move the block into a central function and call it from both `listenSignals` implementations?